containers/compilers.schema.json

```diff
@@ -0,0 +1,21 @@
+{
+    "$schema": "https://json-schema.org/draft/2020-12/schema",
+    "title": "Compilers",
+    "description": "A list of compilers that are supported",
+    "type": "array",
+    "items": {
+        "type": "object",
+        "properties": {
+            "name": {
+                "type": "string"
+            },
+            "package": {
+                "type": "string"
+            },
+            "version": {
+                "type": "string"
+            }
+        },
+        "required": [ "name", "package", "version" ]
```

Comment:
Is it worth specifying `"additionalProperties": false` as well here? So if new fields are added in, the schema validation check will fail so at least it'll act as a reminder to update the schema..

Reply:
Good idea!!